BE/LegalEntities/NorthAmericanEntities/USExampleEntities.rdf

```diff
@@ -0,0 +1,270 @@
+		<dct:license rdf:datatype="&xsd;anyURI">http://opensource.org/licenses/MIT</dct:license>
+		<sm:contentLanguage rdf:datatype="&xsd;anyURI">http://www.w3.org/standards/techs/owl#w3c_all</sm:contentLanguage>
+		<sm:copyright>Copyright (c) 2020 EDM Council, Inc.</sm:copyright>
+		<sm:copyright>Copyright (c) 2020 Object Management Group, Inc.</sm:copyright>
+		<sm:fileAbbreviation>fibo-be-le-usee</sm:fileAbbreviation>
+		<sm:filename>USExampleEntities.rdf</sm:filename>
+		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/BE/Corporations/Corporations/"/>
+		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/BE/GovernmentEntities/NorthAmericanJurisdiction/USGovernmentEntitiesAndJurisdictions/"/>
+		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/BE/LegalEntities/CorporateBodies/"/>
+		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/BE/LegalEntities/LEIEntities/"/>
+		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FND/AgentsAndPeople/Agents/"/>
+		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FND/DatesAndTimes/FinancialDates/"/>
+		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FND/Organizations/FormalOrganizations/"/>
+		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FND/Relations/Relations/"/>
+		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FND/Utilities/AnnotationVocabulary/"/>
+		<owl:imports rdf:resource="https://www.omg.org/spec/LCC/Countries/ISO3166-1-CountryCodes/"/>
+		<owl:versionIRI rdf:resource="https://spec.edmcouncil.org/fibo/ontology/BE/20200101/LegalEntities/NorthAmericanEntities/USExampleEntities/"/>
+		<fibo-fnd-utl-av:hasMaturityLevel rdf:resource="&fibo-fnd-utl-av;Release"/>
+	</owl:Ontology>
+	
+	<owl:NamedIndividual rdf:about="&fibo-be-le-usee;AlphabetInc-US-CA">
+		<rdf:type rdf:resource="&fibo-be-corp-corp;PubliclyHeldCompany"/>
+		<rdf:type rdf:resource="&fibo-be-le-cb;ForProfitCorporation"/>
+		<rdfs:label>Alphabet Inc. US-CA</rdfs:label>
+		<skos:definition>individual representing the Alphabet Inc. legal entity that is a Delaware stock corporation</skos:definition>
+		<fibo-be-corp-corp:hasDateOfIncorporation rdf:resource="&fibo-be-le-usee;AlphabetIncIncorporationDate"/>
+		<fibo-be-le-cb:isIncorporatedIn rdf:resource="&fibo-be-ge-usj;StateOfDelawareJurisdiction"/>
+		<fibo-fnd-org-fm:isDomiciledIn rdf:resource="&lcc-3166-1;UnitedStatesOfAmerica"/>
+		<fibo-fnd-rel-rel:hasLegalName>Alphabet Inc.</fibo-fnd-rel-rel:hasLegalName>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-be-le-usee;AlphabetIncBusinessEntityIdentifier">
+		<rdf:type rdf:resource="&fibo-be-corp-corp;RegistrationIdentifier"/>
+		<rdfs:label>business entity identifier for Alphabet Inc.</rdfs:label>
+		<skos:definition>individual representing the Delaware Division of Corporations business entity identifier for Alphabet Inc.</skos:definition>
```

Comment:
That information is only available in FBC, not in BE.  Thus, the examples have been extended in FBC to add the registration details.

Reply:
fibo-fnd-rel-rel;isDefinedIn &fibo-be-corp-corp;RegistrationIdentifierScheme is a mandatory (some values from) property for &fibo-be-corp-corp;RegistrationIdentifier even in BE.

Regardless, it's pretty fundamental to any identifier what its scope is - so not havign the registry seems to be an abject failure of modularization.

Reply:
The identifier itself, yes is available in BE, and that's where I thought I put it, but the registration entry, which defines who the registration authority is, is in FBC.  This is why I suggested we merge FBC and BE.  It's simply the way it is at the moment.  I did not wan to use this issue to merge FBC and BE.  The failure was due to historical challenges we had in getting anything done in BE or FND.  That isn't an issue now, but the challenge is whether or not we want to totally re-arrange everything now.

Reply:
Even if we stick to only BE there is no reason to miss off the mandatory isDefinedIn &fibo-be-corp-corp;RegistrationIdentifierScheme property from these examples.

Reply:
The business registration identifier scheme was something we missed in the US regulatory agencies ontology.  I'll add individuals for each state for their unique registration scheme in FBC, where the related individuals are defined so that we can then reference it in the FBC extension to these individuals.

Reply:
Done.